src/app/static/src/app/root.ts

```diff
@@ -78,6 +78,14 @@ export interface WarningsState {
     warnings: Warning[]
 }
 
+export enum Steps  {
```

Comment:
Feels like this should be a map of location to step, not an enum mapping step to location (see comment below).